appveyor.yml

```diff
@@ -0,0 +1,34 @@
+# build version format
+version: "{build}"
+
+# fix lineendings in Windows
+init:
+  - git config --global core.autocrlf input
+
+# Test against these versions of Node
+environment:
+  matrix:
+    - nodejs_version: "4"
+    - nodejs_version: "5"
+    - nodejs_version: "6"
+    - nodejs_version: "7"
+    - nodejs_version: "8"
+    - nodejs_version: "9"
+    - nodejs_version: "10"
+
+platform:
+  - x64
+
+install:
+  - ps: Install-Product node $env:nodejs_version $env:platform
+  - npm install
+
+# Post-install test scripts.
+test_script:
+  - node --version
+  - npm --version
+  - npm run dist
+  - powershell .\scripts\build.ps1
+
+# Don't actually build.
```

Comment:
I'm a little confused by the build steps - `dist` is run before `test` (which happens in `build.ps1`), and then there's a comment that says "don't actually build". Is there a way we can make this clearer?

Reply:
I kept it in line with the travis build, but agree it's confusing. `build.ps1` really should be `run_examples.ps1` sort of thing - that's all it does. I might refactor both after the merge.

As for the "Don't actually build" I'll rename to something like "Prevent default msbuild" as that is what Appveyor will try to do

Reply:
I've tidied this up now and think it's more readable - let me know what you think.